web/packages/teleport/src/Bots/Add/GitHubActionsSsh/useGitHubSshFlow.tsx

```diff
@@ -52,7 +51,27 @@ type GitHubFlowContext = {
   resetAttempt: () => void;
 };
 
-const gitHubFlowContext = React.createContext<GitHubFlowContext>(null);
+const noop = () => {};
+const gitHubFlowContext = React.createContext<GitHubFlowContext>({
+  addEmptyRepoRule: noop,
+  attempt: {
+    status: '',
+    statusText: undefined,
+    statusCode: undefined,
+  },
+  createBotRequest: {
+    botName: '',
+    labels: [],
```

Comment:
i was just curious why this was needed? i noticed we don't do this in other places and thinking perhaps we should now